### PR TITLE
Update quota request doc

### PR DESCRIPTION
**Pull request by ShellyXueHan**

### CODE REVIEW

**Comment by mitovskaol** [comment]
@ShellyXueHan We have [a more recent page](https://developer.gov.bc.ca/Need-more-quota-for-OpenShift-project-set) that describes the quota increase project  so I suggest we delete this one.

**Comment by ShellyXueHan** [comment]
@mitovskaol do you mean another page? the one you linked is the same document here.

**Comment by mitovskaol** [comment]
> @mitovskaol do you mean another page? the one you linked is the same document here.

arghh,  I got confused :) Can you please remove the reference to the small, medium and large quota name , and say that all quota increase need an approval by PS team.

File: docs/resourceTuning/request-for-quota-increase.md [hunk 1]
```diff
@@ -30,11 +30,9 @@ Begin a monitoring journey with Sysdig, where you can access dashboards showing
...
 
-**We currently do not support quota increases for a specific resource (i.e. just RAM, or just CPU, or just Storage) or for a specific namespace (i.e. prod only).**  Teams requiring more resources in any of the 3 resource categories such as CPU/RAM/Storage in any of the 4 namespaces (dev, test, tool or prod), will have to submit a standard quota increase request through the Project Registry.  The upgrade path will only be available in this order: from small -> medium -> large. You cannot skip medium and upgrade from small to large.
+Teams requiring more resources in any of the 3 resource categories such as CPU/RAM/Storage in any of the 4 namespaces (dev, test, tool or prod), will have to submit a standard quota increase request through the Project Registry.  The upgrade path will only be available in this order: from small -> medium -> large. You cannot skip medium and upgrade from small to large.
```
Comment (mitovskaol) [inline docs/resourceTuning/request-for-quota-increase.md:33]:
```suggestion
Teams requiring more resources in any of the 3 resource categories such as CPU/RAM/Storage in any of the 4 namespaces (dev, test, tool or prod), will have to submit a standard quota increase request through the Project Registry.  The upgrade path is available to any bigger size quota [LINK TO QUOTA DEFINITION PAGE].
```

**Review by mitovskaol — APPROVED** [review]

File: docs/resourceTuning/request-for-quota-increase.md (hunk 1 above)
Comment (ShellyXueHan) [reply docs/resourceTuning/request-for-quota-increase.md]: got it. updated!

**Pull request merged**